docs/wiki/framework/fortify-framework.md

```diff
@@ -63,6 +64,10 @@ class ACEX_Fortify_Presets {
             {"Bunker", 50}
         };
     };
+    class TAG_categories{
+        displayName = "Combo";
+        presets[] = {"big", "bigGreen"}; // will load presets big and bigGreen each under their own sub category
+    };
```

Comment:
added note that presets can be from configFile as well, which is where `big` comes from
I can add a fuller example if we think that would help

Reply:
I think fuller/additonal example would be better, also can `objects` be mixed with `presets`?

Reply:
yeah they can be mixed in, they'll just be at top level along the sub-categories
I thought `presets[]` was a neat way to organize 
but if it's recursion is confusing to users we can just as easily change to something like
```
class ACEX_Fortify_Presets {
    class TAG_MyPreset2 {
        displayName = "My Preset";
        objects[] = {
            {"Sandbag", 5, "categoryA"},
            {"Bunker", 50, "categoryB"}
```

Reply:
> class ACEX_Fortify_Presets {
>     class TAG_MyPreset2 {
>         displayName = "My Preset";
>         objects[] = {
>             {"Sandbag", 5, "categoryA"},
>             {"Bunker", 50, "categoryB"}
> ```

First and foremost, thank you for your efforts. This, I believe, is the best way to add new categories with the descriptions. ext , when do you think this will be available? 